gpii/configs/gpii.config.untrusted.development.local.txt

```diff
@@ -0,0 +1,21 @@
+gpii.config.untrusted.development.local.json
+============================================
+
+This configuration runs the entire system locally under the hybrid Flow Manager architecture and in development mode.
```

Comment:
This description is identical to the one for gpii.config.untrusted.development.all.local.json, which still seems to exist - perhaps we just need just one of these configs?

Reply:
Given this is a new config, and if you are keeping it, please make it in the json5 style

Reply:
Shall all config files be converted to JSON5 files instead of new ones?

Reply:
I wonder if all config files are converted to JSON5, would it affect other repos such as gpii/windows or ops repos? I checked gpii-ops/gpii-infra repo, which seems ok because it refers to the config file without the file extension part, for example: [gpii.config.cloudBased.flowManager.production](https://github.com/gpii-ops/gpii-infra/blob/master/aws/modules/deploy/83-flowmanager-deploy.erb#L20).

Reply:
As this branch has touched or renamed or created a good number of config files, other repos will have to make corresponding adjustments anyway, I'm thinking we probably should take the chance to convert all config files instead of only new ones. What do you guys think?

Reply:
+1